src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_RANDOM_NUMBER = 20;
+    private static final double DEFAULT_RADIUS = 10.0;
+    private Color randomColor = new ColorSupplier().getRandomColor();
```

Comment:
you should make ColorSupplier global, not Color, color shouldn't be the same for all random figures